cc3d/core/PyCoreSpecs.py

```diff
@@ -1543,7 +1543,7 @@ def xml(self) -> ElementCC3D:
                 fa_el.ElementCC3D("FluctuationAmplitudeParameters", {"CellType": _type,
                                                                      "FluctuationAmplitude": str(_fa)})
         else:
-            self._el.ElementCC3D("FluctuationAmplitude", {}, str(self.spec_dict["fluctuation_amplitude"]))
+            self._el.ElementCC3D("Temperature", {}, str(self.spec_dict["fluctuation_amplitude"]))
```

Comment:
If neither FluctuationAmplitude nor Temperature is set, we default to 0.0, right?

Reply:
In the Python API, a value is always set. The default value is 10 as a default argument, per recommendations in the code snippets.